engine/src/main/java/org/hibernate/search/store/impl/FSSlaveDirectoryProvider.java

```diff
@@ -76,16 +78,11 @@ public void initialize(String directoryProviderName, Properties properties, Buil
 		this.directoryProviderName = directoryProviderName;
 		this.serviceManager = context.getServiceManager();
 		//source guessing
-		sourceIndexDir = DirectoryProviderHelper.getSourceDirectory( directoryProviderName, properties, false );
-		log.debugf( "Source directory: %s", sourceIndexDir.getPath() );
-		indexDir = DirectoryHelper.getVerifiedIndexDir( directoryProviderName, properties, true );
-		log.debugf( "Index directory: %s", indexDir.getPath() );
-		try {
-			indexName = indexDir.getCanonicalPath();
-		}
-		catch (IOException e) {
-			throw new SearchException( "Unable to initialize index: " + directoryProviderName, e );
-		}
+		sourceIndexDir = DirectoryProviderHelper.getSourceDirectoryPath( directoryProviderName, properties, false );
+		log.debugf( "Source directory: %s", sourceIndexDir );
+		indexDir = DirectoryHelper.getVerifiedIndexPath( directoryProviderName, properties, true );
```

Comment:
Same remark as in `FSMasterDirectoryProvider.java` about normalizing `indexDir`.